Comanche055/GROUND_TRACKING_DETERMINATION_PROGRAM.agc

```diff
@@ -29,59 +29,61 @@
 # Please report any errors to info@sandroid.org.
 
 # Page 456
-# GROUND TRACKING DETERMINATION PROGRAM -- P21
-#
+# GROUND TRACKING DETERMINATION PROGRAM P21
 # PROGRAM DESCRIPTION
-#	MOD NO -- 1
-#	MOD BY -- N. M. NEVILLE
-#
-# FUNCTIONAL DESCRIPTION --
-#	TO PROVIDE THE ASTRONAUT DETAILS OF THE LM OR CSM GROUND TRACK WITHOUT
-#	THE NEED FOR GROUND COMMUNICATION (REQUESTED BY DSKY).
-#
-# CALLING SEQUENCE --
-#	ASTRONAUT REQUEST THROUGH DSKY V37E21E
-#
-# SUBROUTINES CALLED --
-#	GOPERF4
-#	GOFLASH
-#	THISPREC
-#	OTHPREC
-#	LAT-LONG
-#
-# NORMAL EXIT MODES --
-#	ASTRONAUT REQUEST THROUGH DSKY TO TERMINATE PROGRAM V34E
-#
-# ALARM OR ABORT EXIT MODES --
-#	NONE
-#
-# OUTPUT --
-#	OCTAL DISPLAY OF OPTION CODE AND VEHICLE WHOSE GROUND TRACK IS TO BE
-#	COMPUTED
-#		OPTION CODE	00002
-#		THIS		00001
-#		OTHER		00002
-#	DECIMAL DISPLAY OF TIME TO BE INTEGRATED TO HOURS , MINUTES , SECONDS
-#	DECIMAL DISPLAY OF LAT,LONG,ALT
-#
+# MOD NO - 1
+# MOD BY - N.M.NEVILLE
+# FUNCTIONAL DECRIPTION-
+#
+# TO PROVIDE THE ASTRONAUT DETAILS OF THE LM OR CSM GROUND TRACK WITHOUT
+# THE NEED FOR GROUND COMMUNICATION (REQUESTED BY DSKY).
+# CALLING SEQUENCE -
+#
+# ASTRONAUT REQUEST THROUGH DSKY V37E21E
+# SUBROUTINES CALLED-
+#
+# GOPERF4
+# GOFLASH
+# THISPREC
+# OTHPREC
+# LAT-LONG
+# NORMAL EXIT MODES-
+#
+# ASTRONAUT REQUEST THROUGH DSKY TO TERMINATE PROGRAM V34E
+# ALARM OR ABORT EXIT MODES-
+#
+# NONE
+# OUTPUT -
+#
+# OCTAL DISPLAY OF OPTION CODE AND VEHICLE WHOSE GROUND TRACK IS TO BE
+# COMPUTED
+#	OPTION CODE	00002
+#	THIS		00001
+#	OTHER		00002
+# DECIMAL DISPLAY OF TIME TO BE INTEGRATED TO HOURS , MINUTES , SECONDS
+# DECIMAL DISPLAY OF LAT,LONG,ALT
 # ERASABLE INITIALIZATION REQUIRED
-#	AX0	 2DEC	4.652459653 E-5   RADIANS       "68-69 CONSTANTS"
-#	-AY0	 2DEC	2.147535898 E-5   RADIANS
-#	AZ0	 2DEC	.7753206164	  REVOLUTIONS
-# 	FOR LUNAR ORBITS 504LM VECTOR IS NEEDED
-#	504LM	 2DEC	-2.700340600 E-5  RADIANS
-#	504LM _2 2DEC	-7.514128400 E-4  RADIANS
-#	504LM _4 2DEC	_2.553198641 E-4  RADIANS
 #
-#	NONE
+# AX0		2DEC	4.652459653 E-5		RADIANS		"68-69 CONSTANTS"
```

Comment:
For this block (till line 80: `NONE`), I'm actually not really sure which belongs to which column 😅